src/MetadataUtility/Audio/Wave.cs

```diff
@@ -2,12 +2,156 @@
 // All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group.
 // </copyright>
 
-namespace MetadataUtility.Audio.Vendors
+namespace MetadataUtility.Audio
 {
+    using LanguageExt;
+    using LanguageExt.Common;
+    using MetadataUtility.Utilities;
+
     public static class Wave
     {
+        public const string Mime = "audio/wave";
+
+        public const int WaveFileOffset = 7;
```

Comment:
This name is a little ambiguous to me. The offset of what? Also `7` is a very suspicious number, usually offsets are even multiples.

Reply:
I'm looking at this: http://soundfile.sapp.org/doc/WaveFormat/

The offset for the magic number? 

You'll also want to check for the `RIFF` magic number at the start of the file.